File: docs/src/types/boolean/bool_2.py
if __name__ == "__main__":
    if True:
        print("if True")  # always works code

    if False:
        print("if False")  # unreachable code

    if not True:
        print("if not True")  # unreachable code

    if not False:
        print("if not False")  # always works code

    if True & False:
        print("if True & False")  # unreachable code

    if True & True:
        print("if True & True")  # always works code

    if False & False:
        print("if False & False")  # unreachable code

    if True | False:
        print("if True | False")  # always works code

    if True | True:
        print("if True | True")  # always works code

    if False | False:
        print("if False | False")  # unreachable code

File: docs/src/types/boolean/bool_2.rs
fn main() {
    if true {
        println!("if true"); // always works code
    }

    if false {
        println!("if false"); // unreachable code
    }

    if !true {
        println!("if !true"); // unreachable code
    }

    if !false {
        println!("if !false"); // always works code
    }

    if true & false {
        println!("if true & false"); // unreachable code
    }

    if true & true {
        println!("if true & true"); // always works code
    }

    if false & false {
        println!("if false & false"); // unreachable code
    }

    if true | false {
        println!("if true | false"); // always works code
    }

    if true | true {
        println!("if true | true"); // always works code
    }

    if false | false {
        println!("if false | false"); // unreachable code
    }
}
